xsl-test/templates/project.xsl

```diff
@@ -23,8 +23,9 @@ SOFTWARE.
       <xsl:with-param name="message" select="'Fails to format project name'"/>
       <xsl:with-param name="expected">
         <span>
-          <code>
-            <a href="https://www.0crat.com/p/ABCDEFGHI" title="Project ABCDEFGHI">
+          <code style="background-color:#daf1e0;">
```

Comment:
@paulodamaso Why the ``background-color`` attribute, what does it do? And is the reindentation of ``<a>`` needed?

Reply:
@amihaiemil This bug correction depends on https://github.com/zerocracy/datum/issues/339; I've had to put it here so the build doesn't fali.